helm/vertical-pod-autoscaler-app/templates/ciliumnetworkpolicy.yaml

```diff
@@ -0,0 +1,16 @@
+{{ if .Values.ciliumNetworkPolicy.enabled }}
```

Comment:
what's the issue with keeping the capabilities check? I'd rather we have both

Reply:
If you add the capabilities check, if this app is installed before cilium in a cluster then installation will succeed, but the cilium network policy will NOT be created, meaning VPA wont work.
If the `ciliumNetworkPolicy.enabled` is set to true, then helm chart creation fails if cilium is not yet installed and that is exactly the desired behavior